src/XPurchase/VirtoCommerce.XPurchase/Schemas/GiftItemType.cs

```diff
@@ -0,0 +1,56 @@
+using System.Linq;
+using GraphQL.DataLoader;
+using GraphQL.Resolvers;
+using GraphQL.Types;
+using MediatR;
+using VirtoCommerce.CartModule.Core.Model;
+using VirtoCommerce.ExperienceApiModule.Core.Extensions;
+using VirtoCommerce.ExperienceApiModule.Core.Helpers;
+using VirtoCommerce.ExperienceApiModule.Core.Schemas;
+using VirtoCommerce.ExperienceApiModule.Core.Services;
+using VirtoCommerce.XDigitalCatalog;
+using VirtoCommerce.XDigitalCatalog.Queries;
+using VirtoCommerce.XDigitalCatalog.Schemas;
+
+namespace VirtoCommerce.XPurchase.Schemas
+{
+    public class GiftItemType : ExtendableGraphType<LineItem>
```

Comment:
I think we can do not inherit GiftItemType from LineItemType in order to do not confuse extra  properties such prices and discounts  that don't make sense for gift